functions/AutoRemediateCT-001.js

```diff
@@ -0,0 +1,47 @@
+"use strict";
+
+const AWS = require("aws-sdk");
+
+/**
+* Lambda function to automatically remediate CloudTrail not Enabled
+*/
+module.exports.handler = (event, context, callback) => {
+
+	console.log('CloudTrail not Global - Received event:', JSON.stringify(event, null, 2));
+
+	if (!event || !event.region || event.ruleId !== "CT-001") {
+		return handleError("Invalid event");
+	}
+
+	var cloudtrail = new AWS.CloudTrail({apiVersion: '2013-11-01'});
+	cloudtrail.updateTrail({
+
+		Name: 'GlobalTrail',
+		S3BucketName: 'cc-remediate-cloudtrail',
+		// CloudWatchLogsLogGroupArn: 'STRING_VALUE',
+		// CloudWatchLogsRoleArn: 'STRING_VALUE',
+		IncludeGlobalServiceEvents: true,
```

Comment:
This will break another rule [GlobalServicesLoggingDuplicated](https://www.cloudconformity.com/conformity-rules/CloudTrail/cloudtrail-global-services-logging-duplicated.html). I'd suggest enable this only for us-east-1 
Have a look at https://github.com/cloudconformity/auto-remediate/blob/master/functions/AutoRemediateConfig-001.js (Line 189)

Reply:
I attempted to implement this, and received this error:
"2017-11-15T01:06:45.283Z 3ebd24a9-c9a1-11e7-b120-ab9e4717955c ErrorInvalidParameterCombinationException: Multi-Region trail must include global service events."
It appears the API requires this regardless?